Amazon.IonObjectMapper.Test/FactoryTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿/*
+ * Copyright (c) Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+namespace Amazon.IonObjectMapper.Test
+{
+    using System;
+    using System.IO;
+    using Amazon.IonDotnet.Tree;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+    [TestClass]
+    public class FactoryTest
```

Comment:
What's the difference between this test and `Amazon.IonObjectMapper.Demo/NamingConventionTest.cs`?

Reply:
Nothing 😅 - I copied that test to the Demo module for completeness

Reply:
Should we delete this test then? What's the purpose to add the FactoryTest to the Test module while we already have a `NamingConventionTest` in it https://github.com/amzn/ion-object-mapper-dotnet/blob/main/Amazon.IonObjectMapper.Test/NamingConventionTest.cs?

Reply:
Deleted `FactoryTest` from the Test module